### DISCUSSION TITLE
Welcome to coteafs-selenium Discussions!

### DISCUSSION THREAD

**Discussion by WasiqB** [WasiqB/coteafs-selenium#146] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chameleonlabs2021** [comment]
My name is Rahul Poolanchalil, and I am reaching out to express my keen interest in contributing to the   coteafs-selenium project. As an admirer of open-source initiatives and a professional with a background in data engineering, I am eager to lend my skills and enthusiasm to further enhance the capabilities of selenium.

I possess a strong foundation in Python programming. I am particularly interested in improving documentation, developing new features, fixing bugs etc. I believe that my background can contribute to the ongoing efforts.

**Reply by mfaisalkhatri** [reply]
@chameleonlabs2021 This project is no longer maintained, please checkout https://github.com/BoykaFramework/boyka-framework
